packages/parser/lib/processProps.ts

```diff
@@ -35,15 +35,22 @@ export function processPropValue(propValueNode: bt.Node, result: PropsResult) {
         result.typeDesc = typeDesc
       }
     }
-
+    // Processing props's default value
     otherNodes.forEach((node: any) => {
       const n = node.key.name
       if (n === 'default') {
-        if (!hasFunctionTypeDef(result.type) && bt.isFunction(node.value)) {
```

Comment:
When a prop type is function , we should not execute this function, for example:
```js
{
   propA: {
      type: Function,
      default: function () {
          // ......
      }
    }
}
```
Because the type of propA is a function, the default value should be the function itself, not the result of the function.

So this test case is incorrect: https://github.com/vue-contrib/vuese/pull/106/files#diff-769dd406bade87c948da829f10edee1dL243

Reply:
Thank you very much for your great work. 🤟